docs/detections/api/exceptions/api-create-exception-item.asciidoc

```diff
@@ -43,7 +43,7 @@ A JSON object with these fields:
 exception queries. Boolean `AND` logic is used to evaluate the relationship
 between array elements. If you want to use `OR` logic, create a separate
 exception item. |Yes
-|`expire_time` |String |An expiration date in ISO format. |No
+|`expire_time` |String |An expiration date in ISO format. This field is not available for endpoint exceptions. |No
```

Comment:
Users can likely infer that this param only works for regular exceptions, but it might be better to directly state that and remove any potential for ambiguity. 

  ```suggestion
|`expire_time` |String a|Specify when an exception item expires. The expiration date must be in ISO format. 
NOTE: This field is only available for regular exception items, not endpoint exceptions. |No
```

Reply:
Thanks, @nastasha-solomon! I agree it's better to be explicit in this case – I'll make that change.

For the sake of consistency, I'd slightly tweak your suggestion for the first sentence, since most of the other parameter descriptions start with a noun phrase rather than an imperative verb. How does this sound:

`The exception item's expiration date, in ISO format. This field is only available for regular exception items, not endpoint exceptions.`